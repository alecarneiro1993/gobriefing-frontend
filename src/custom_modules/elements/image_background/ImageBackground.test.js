import React from "react";
import { shallow } from "enzyme";

import { ImageBackground } from ".";

describe("ImageBackground", () => {
  it("renders without crashing", () => {
    const src =
      "https://dslv9ilpbe7p1.cloudfront.net/hownJrwXlTVdURCWN3Sj1g_store_header_image";
    const wrapper = shallow(<ImageBackground src={src} />);
    expect(wrapper).toMatchSnapshot();
  });
});
