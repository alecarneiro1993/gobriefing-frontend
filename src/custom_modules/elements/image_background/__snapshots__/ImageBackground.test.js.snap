// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageBackground renders without crashing 1`] = `
<ImageBackgroundWrapper
  color=""
  src="https://dslv9ilpbe7p1.cloudfront.net/hownJrwXlTVdURCWN3Sj1g_store_header_image"
/>
`;
